src/DL/TransactionHistoryHandler.java

```diff
@@ -0,0 +1,77 @@
+package DL;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
+public class TransactionHistoryHandler {
+
+    private final String path;
```

Comment:
no need to store path in class variables. it is better to have File variable and init it in constructor

Reply:
https://github.com/ychepel/currency-exchange/pull/4#discussion_r1470028315